docs/content/usage-reporting.md

```diff
@@ -0,0 +1,222 @@
+          "max_waf_count": 7,
+          "max_dos_count": 1
+        }
+      }
+    },
+    {
+      "metadata": {
+        "displayName": "my-cluster2",
+        "uid": "12tgb8ug-g8ik-bs7h-gj3j-hjitk672946hb",
+        "createTime": "2023-01-25T09:12:33.001Z",
+        "updateTime": "2023-01-26T10:12:33.001Z",
+        "monthReturned": "May"
+      },
+      "node_count": 3,
+      "max_node_count": 3,
+      "pod_details": {
+        "current_pod_counts": {
+          "pod_count": 5,
+          "waf_count": 5,
+          "dos_count": 0
+        },
+        "max_pod_counts": {
+          "max_pod_count": 15,
+          "max_waf_count": 5,
+          "max_dos_count": 0
+        }
+      }
+    }
+  ]
+}
+```
+If you want a friendly name for each cluster in the response, You can specify the `displayName` for the cluster with the `-cluster-display-name` command-line argument when you deploy the Cluster Connector . See [Command-line Arguments](#Command-line Arguments) for more information. From this response, you can see the cluster uid corresponding to the cluster name.
+
+You can also query the usage data for a specific cluster by specifying the cluster uid in the endpoint, for example:
+```
+> curl --user "foo:bar" https://nms.example.com/api/platform/v1/k8s-usage/d290f1ee-6c54-4b01-90e6-d701748f0851
+{
+  "metadata": {
+    "displayName": "my-cluster",
+    "uid": "d290f1ee-6c54-4b01-90e6-d701748f0851",
+    "createTime": "2023-01-27T09:12:33.001Z",
+    "updateTime": "2023-01-29T10:12:33.001Z",
+    "monthReturned": "May"
+  },
+  "node_count": 4,
+  "max_node_count": 5,
+  "pod_details": {
+    "current_pod_counts": {
+      "pod_count": 15,
+      "waf_count": 5,
+      "dos_count": 0
+    },
+    "max_pod_counts": {
+      "max_pod_count": 25,
+      "max_waf_count": 7,
+      "max_dos_count": 1
+    }
+  }
+}
+```
+
+## Uninstalling the NGINX Cluster Connector
+To remove the Nginx Cluster Connector from your Kubernetes cluster, run the following command:
+```
+kubectl delete -f cluster-connector.yaml
+```
+
+## Command-line Arguments
+The NGINX Cluster Connector supports several command-line arguments. The command-line arguments can be specified in the `args` section of the Kubernetes deployment file. The following is a list of the supported command-line arguments and their usage:
+
+### -nms-server-address `<string>`
+The address of the NGINX Management Suite host. IPv4 addresses and hostnames are supported.
+Default `http://apigw.nms.svc.cluster.local/api/platform/v1/k8s-usage`.
+
+### -nms-basic-auth-secret `<string>`
+Secret for basic authentication to the NGINX Management Suite API. The secret must be in `kubernetes.io/basic-auth` format using base64 encoding.
+Format `<namespace>/<name>`.
+
+### -cluster-display-name `<string>`
+The display name of the Kubernetes cluster.
+
+### -skip-tls-verify
+Skip TLS verification for the NMS server. **For testing purposes with NMS server using self-assigned certificate.**
```

Comment:
I would use `NGINX Management Suite` here too and the other two occurrences since you user the full name in this section

Reply:
Will edit it in the next commit